backend/core/routes/routes.go

```diff
@@ -33,9 +33,10 @@ func SetupRoutes(r *gin.Engine) *gin.Engine {
 	base.POST("/user/verify-token", user.VerifyToken)
 	base.GET("/user/me", middleware.Authenticate(), user.GetCurrentUser)
 	base.POST("/user/bind", middleware.Authenticate(), user.BindAdminToCongregation)
+	base.DELETE("/user/logout", middleware.Authenticate(), user.LogoutUser)
```

Comment:
What's the thinking behind using DELETE for the logout endpoint instead of just POST? It makes more sense to me to use POST because we're not actually DELETEing a resource. As this [stackoverflow answer ](https://stackoverflow.com/a/70973031) states it doesn't matter what you use but i was just curious why you decided to do it here for the log out endpoint :)

Reply:
Good point. I wasn't really thinking 😄 POST does make more sense lol I'll change that soon.

Reply:
No problems :)